test/suite/test_verify.py

```diff
@@ -177,19 +194,40 @@ def test_verify_api_read_corrupt_pages(self):
             for i in range(0, 100):
                 f.write(b'\x01\xff\x80')
 
+    def test_verify_api_corrupt_first_page(self):
+        """
+        Test verify via API, on a table that is purposely corrupted in
+        the first page. A verify operation with read_corrupt on should
+        result in a checksum errors being logged.
+        """
+        params = 'key_format=S,value_format=S'
+        self.session.create('table:' + self.tablename, params)
+        self.populate(self.tablename)
+
+        self.runWt(['verify', '-d', 'dump_address', 'table:' + self.tablename, '-d'], outfilename='dump.out')
+
+        # Grab the offset position of the first page.
```

Comment:
Similar to my previous comment I suggest spelling out what's happening here in a bit more detail.
```
# wt verify -d dump_address performs a depth-first traversal of the BTree. So the first leaf page 
# it prints is the first child of its parent. Grab the offset of this one so we can corrupt it.